package.json

```diff
@@ -335,7 +335,7 @@
     "url-search-params-polyfill": "^8.1.0",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#b322bfdae8939ab7dc097625ffb909b7d845cd0f",
+    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema",
```

Comment:
We need to pin the dependency to a specific commit hash to make sure any breaking changes introduced in a new `vets-json-schema` version are resolved in the same PR that upgrades the version. You can update the commit hash to the latest `master` commit with the `yarn update:schema` command from inside `vets-website`.